### PR TITLE
Added Weather Widget UI Design 

### PR DESCRIPTION
**Pull request by Nidhikumari-4**

### CODE REVIEW

File: Cards/Card_01/code/style.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    outline: none;
+}
+html,body{
+    font-family: 'Roboto', sans-serif;
+    overflow: hidden;
+    background: #d22222;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```
Comment (thepranaygupta) [inline Cards/Card_01/code/style.css:13]: use classes instead of tags to style elements

**Review by thepranaygupta — CHANGES_REQUESTED** [review]

**Review by nandini7637 — CHANGES_REQUESTED** [review]
You have added your calendar code, kindly check

**Comment by Nidhikumari-4** [comment]
@nandini7637  I had added calendar code by mistake.
I have made the changes kindly check.

**Review by Abirpal202049 — APPROVED** [review]

**Review by nandini7637 — CHANGES_REQUESTED** [review]
Rename folder as Weather Widget Card, else every thing LGTM

**Comment by Nidhikumari-4** [comment]
@nandini7637 I have renamed the folder, kindly check .

**Review by nandini7637 — APPROVED** [review]

**Pull request merged**